README.md

```diff
@@ -137,6 +138,12 @@ Default value: `misc`
 
 The software category. Used to fill the "section" field of the control file
 
+#### options.extra_headers
+Type: `array<String>`
+Default value: `[]`
+
+Extra headers to add to the package's control file, e.g. `["Foo: 1.2.3", "Bar: abc"]`
```

Comment:
In order to enforce a styling of the headers, I would prefer using an object here. Something like:
```json
{
  "Foo": "1.2.3",
  "Bar": "abc"
}
```
would then be added as: 
```
Foo: 1.2.3
Bar: abc
```
To make sure every configuration respects the `<name>: <value>` format of the control file.
What do you think?